lib/streamlit/elements/deck_gl_json_chart.py

```diff
@@ -108,6 +108,21 @@ def dg(self) -> "streamlit.delta_generator.DeltaGenerator":
 }
 
 
+def _get_pydesk_tooltip(pydeck_obj) -> Optional[Dict[str, str]]:
```

Comment:
nit: Looks like there was a typo where `pydesk` was written instead of `pydeck` in a few places in this PR

Reply:
Fixed typo. On Monday, I will add tests for the _get_pydesk_tooltip method